src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -6410,6 +6410,27 @@ void AuraEffect::HandleBattlegroundPlayerPosition(AuraApplication const* aurApp,
         bg->RemovePlayerPosition(target->GetGUID());
 }
 
+void AuraEffect::HandleModArmorPctFromStat(AuraApplication const* aurApp, uint8 mode, bool apply) const
+{
+    // Increase armor by <AuraEffect.BasePoints> % of your <primary stat>
+
```

Comment:
I could be wrong, but it looks like this blank line is redundant.
```suggestion
```

Reply:
I wanted to make sure the comment isn't assosiated with the line after it but rather with the entire body;
Any better suggestions than this empty newline?

Reply:
Aha, I see what you mean. So the comment does not belong to the next line.
How about outside the brackets, either right before or right after `void AuraEffect:: [...]` ?

Reply:
Haven't seen any comments outside methods but I do like it;
Moved that code to a different PR, but I'll just do that there.